src/main/resources/config/modules/Vanilla.xml

```diff
@@ -726,9 +726,9 @@
             <BiomeType name='Swamp'  />
             <BiomeType name='Beach'  />
             <BiomeType name='River'  />
-            <Biome name='Ocean'  weight='-1' />
-            <Biome name='Desert'  weight='-1' />
-            <Biome name='Mountain'  weight='-1' />
+            <Biome name='.*Ocean.*'  weight='-1' />
```

Comment:
Looking at the file, that's only true of `vnlaClayVeins`, not `vnlaClayPreferredVeins`.